sklearn/multiclass.py

```diff
@@ -160,7 +160,9 @@ class OneVsRestClassifier(MultiOutputMixin, ClassifierMixin,
         :term:`decision_function` or :term:`predict_proba`.
 
     n_jobs : int or None, optional (default=None)
-        The number of jobs to use for the computation.
+        The number of jobs to use for the computation, where each job computes
+        a classifier in parallel.
```

Comment:
thanks for the PR @annejeevan 

I feel like the current formulation may suggest that the internal implementation of the classifier is parallelized which is usually not the case

How about

> The number of jobs to use for the computation: the `n_classes` one-vs-rest problems are computed in parallel.

Reply:
Thank you for the feedback @NicolasHug . 
Yes, it looks like. Sure. That looks good. I will try to modify the doc string.

Reply:
Hey @NicolasHug, I have modified accordingly. Can you please check the PR? Thanks!